crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs

```diff
@@ -140,6 +141,17 @@ pub fn execute_call_entry_point(
     };
     runtime_state.cheatnet_state.trace_data.exit_nested_call(
         resources,
+        result.as_ref().map_or_else(
+            |_| vec![],
+            |call_info| {
+                call_info
+                    .execution
+                    .l2_to_l1_messages
+                    .iter()
+                    .map(|ordered_message| ordered_message.message.payload.0.len())
+                    .collect()
+            },
```

Comment:
Separate var pls 🙏

Reply:
can we pass `call_info.execution.l2_to_l1_messages` instead and process it down the line? it makes code less readable here